libnmstate/nm/applier.py

```diff
@@ -85,23 +86,73 @@ def prepare_edited_ifaces_configuration(ifaces_desired_state):
     return con_profiles
 
 
-def set_ifaces_admin_state(ifaces_desired_state):
+def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
+    """
+    Control interface admin state by activating, deactivating and deleting
+    devices connection profiles.
+
+    The `down` and `absent` states are handled in the same manner (temporary)
```

Comment:
How about:
```
The `absent` state results in deactivating the device and deleting the connection profile.
FIXME: The `down` state is currently handled in the same way.
```

This makes it more clear that the down state may change in the future.